config.go

```diff
@@ -376,3 +376,13 @@ func loadConfiguration(configPath string, ignoreLogging bool) (resolvedConfigPat
 
 	return resolved, logfilePath, config, nil
 }
+
+func ociResourceUpdateRuntimeConfig(ocispec oci.CompatOCISpec, config *oci.RuntimeConfig) error {
+	mem := int(float64(*ocispec.Linux.Resources.Memory.Limit) / 1024 / 1024)
```

Comment:
@grahamwhaley @jodh-intel 

Right now, I don't think we should arbitrarily add 25M by default. I run the following tests.

I use `docker run  -m MEMORY centos:7 /bin/free -m` to test

128m
```
              total        used        free      shared  buff/cache   available
Mem:            117           9          86           0          21          91
```

512m
```
              total        used        free      shared  buff/cache   available
Mem:            494          12         461           0          20         461
```

1024m
```
              total        used        free      shared  buff/cache   available
Mem:            998          13         964           0          20         959
```

2048m
```
              total        used        free      shared  buff/cache   available
Mem:           2004          12        1971           0          20        1960
```

4096m
```
              total        used        free      shared  buff/cache   available
Mem:           4017          18        3980           0          18        3949
```

I find the kernel memory footprint is not consistent. And looking at total memory, some of them is eaten by firmware(?).

It's not easy to get available memory to precisely what user requested. And if we add extra memory for kernel overhead say 25M with requested 128M, we would ended up with total memory 140M. eg:
```
# docker run -m 128m centos:7 /bin/free -m
              total        used        free      shared  buff/cache   available
Mem:            140           9         109           0          20         114
```

It looks pretty strange and confused to me, since I only requested 128M memory in the first  place.

From my point of view, I can expect less memory available to my application than I request. Because I should know clear container is using VM and VM kernel consumes memory.

I think adding an extra to requested memory should rethink (carefully). 

For the time being (for the sake of this patch), we can focus on adding this feature as first step. Then think about how we handle VM kernel memory footprint (documentation or extra memory ...)